DogsList/DogsList/SlideMenu/ContainerViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ContainerViewController.swift
+//  DogsList
+//
+//  Created by Nick Krasnitskiy on 18.08.2021.
+//
+
+import UIKit
+
+class ContainerViewController: UIViewController {
+    
+    @IBOutlet private weak var scrollView: UIScrollView!
+    let leftMenuWidth:CGFloat = 260
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        DispatchQueue.main.async() {
+            self.closeMenu(animated: false)
+        }
+                
+        NotificationCenter.default.addObserver(self, selector: #selector(ContainerViewController.toggleMenu), name: NSNotification.Name(rawValue: "toggleMenu"), object: nil)
```

Comment:
try to use Swift api instead of ObjectiveC if possible

Reply:
Ok. Fixed.